src/legacy/status_im/data_store/messages.cljs

```diff
@@ -16,6 +16,30 @@
                       :community-id      :communityId
                       :clock-value       :clock})))
 
+(defn- <-status-link-previews-rpc
```

Comment:
whu this one is needed?

Reply:
To convert the object of the received message from camelCase to kebab-case.